config/playground/demo/observability/otel_demo/nginx.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+# Nginx Deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: otel-demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:latest
+          ports:
+            - containerPort: 90
+          volumeMounts:
+            - name: config-volume
+              mountPath: /etc/nginx/conf.d/default.conf
+              subPath: default.conf
+      volumes:
+        - name: config-volume
+          configMap:
+            name: nginx-config
+            items:
+              - key: default.conf
+                path: default.conf
+        - name: opentelemetry-volume
+          configMap:
+            name: nginx-config
+            items:
+              - key: opentelemetry_module.conf
+                path: opentelemetry_module.conf
+---
+# Nginx Config-map
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-config
+  namespace: otel-demo
+data:
+  default.conf: |
+    server {
+      listen 90;
+
+      location / {
+         proxy_pass http://frontend:8080/;
```

Comment:
fronted didn't works, and make container crashed as cannot resolve http://frontend. I changed localhost works.

````
proxy_pass http://localhost:8080/;
````